src/app/data-planes/views/DataPlaneListView.vue

```diff
@@ -1,157 +1,89 @@
-  selectedDppName: {
-    type: String,
-    required: false,
-    default: null,
-  },
-
-  gatewayType: {
-    type: String,
-    required: false,
-    default: 'true',
-  },
-
-  offset: {
-    type: Number,
-    required: false,
-    default: 0,
-  },
-
-  isGatewayView: {
-    type: Boolean,
-    required: false,
-    default: false,
-  },
-})
-
-const dataPlaneOverviews = ref<DataPlaneOverview[]>([])
-
-const isLoading = ref(true)
-const error = ref<Error | null>(null)
-const nextUrl = ref<string | null>(null)
-const pageOffset = ref(props.offset)
-const dppFilterFields = computed(() => {
-  return {
-    ...BASE_FILTER_FIELDS,
-    ...(props.isGatewayView ? GATEWAY_FILTER_FIELDS : DPP_FILTER_FIELDS),
-  }
-})
-
-function start() {
-  const filterFields = QueryParameter.get('filterFields')
-  const dppParams = filterFields !== null ? JSON.parse(filterFields) as DataPlaneOverviewParameters : {}
-
-  loadData(props.offset, {
-    ...dppParams,
-    gateway: props.gatewayType,
-  } as DataPlaneOverviewParameters)
-}
-
-start()
-
-async function loadData(offset: number, dppParams: DataPlaneOverviewParameters = {}): Promise<void> {
-  pageOffset.value = offset
-  // Puts the offset parameter in the URL so it can be retrieved when the user reloads the page.
-  QueryParameter.set('offset', offset > 0 ? offset : null)
-  QueryParameter.set('gatewayType', dppParams.gateway === 'true' ? 'all' : dppParams.gateway)
-  isLoading.value = true
-
-  const mesh = route.params.mesh as string
-  const params = getDataplaneOverviewParameters(dppParams, PAGE_SIZE_DEFAULT, offset, props.isGatewayView)
-
-  try {
-    const { items, next } = await kumaApi.getAllDataplaneOverviewsFromMesh({ mesh }, params)
-
-    nextUrl.value = next
-    dataPlaneOverviews.value = items ?? []
-  } catch (err) {
-    if (err instanceof Error) {
-      error.value = err
-    } else {
-      console.error(err)
-    }
-
-    dataPlaneOverviews.value = []
-    nextUrl.value = null
-  } finally {
-    isLoading.value = false
-  }
-}
-
-function getDataplaneOverviewParameters(dppParams: DataPlaneOverviewParameters, size: number, offset: number, isGatewayView: boolean): DataPlaneOverviewParameters {
-  const params: DataPlaneOverviewParameters = {
-    ...dppParams,
-    size,
-    offset,
-  }
-
-  if (isGatewayView && (!('gateway' in params) || params.gateway === 'false')) {
-    params.gateway = 'true'
-  } else if (!isGatewayView) {
-    params.gateway = 'false'
-  }
-
-  return params
-}
+const props = defineProps<{
+  page: number
+  size: number
+  search: string
+  query: string
+  //
+  mesh: string
+}>()
 </script>
-
-<style lang="scss">
-.table-header-selector-item .k-dropdown-item-trigger {
-  // Removes the paddings from the dropdown items so that the items are interactive for their full visual size.
-  padding: 0 !important;
+<style lang="scss" scoped>
+.data-plane-proxy-filter {
+  flex-basis: 350px;
+  flex-grow: 1;
+  margin-right: auto;
 }
 </style>
```

Comment:
**Suggestion**: Can you make sure to add the same kinds of styles I added in the already converted list views for column sizing?

- Would need the following styles for three tables
- Would need the class name for the three tables

```suggestion
.actions-dropdown {
  display: inline-block;
}
</style>

<style lang="scss">
.data-plane-proxy-collection {
  .actions-column {
    width: 5%;
    min-width: 80px;
    text-align: end;
  }

  .status-column {
    width: 10%;
    min-width: 200px;
  }
}
</style>
```

Reply:
Oh yes good call absolutely! Will look now